### PR TITLE
[DOCFIX]Update docs/en/Running-Alluxio-Fault-Tolerant.md

### PR DESCRIPTION
**Pull request by MovingLi**

### CODE REVIEW

**Comment by saltylin** [comment]
@MovingLi LGTM. Thanks very much for your translation!!

File: docs/cn/Running-Alluxio-Fault-Tolerant.md [hunk 1]
```diff
@@ -83,11 +83,11 @@ Zookeeper和共享文件系统都正常运行时，需要在每个主机上配
 
 ### Master配置
 
-除了以上配置，Alluxio master需要额外的配置，以下变量需在`conf/alluxio-env.sh`中正确设置：
+除了以上配置，Alluxio master需要额外的配置，以下变量需在`conf/alluxio-site.properties`中正确设置：
 
-    export ALLUXIO_MASTER_HOSTNAME=[externally visible address of this machine]
+    export alluxio.master.hostname=[externally visible address of this machine]
```
Comment (apc999) [inline docs/cn/Running-Alluxio-Fault-Tolerant.md]: remove `export`

File: docs/cn/Running-Alluxio-Fault-Tolerant.md [hunk 2]
```diff
@@ -97,7 +97,7 @@ Zookeeper和共享文件系统都正常运行时，需要在每个主机上配
 
 ### Worker配置
 
-只要以上参数配置正确，worker就可以咨询ZooKeeper，找到当前应当连接的master。所以，worker无需设置`ALLUXIO_MASTER_HOSTNAME`。
+只要以上参数配置正确，worker就可以咨询ZooKeeper，找到当前应当连接的master。所以，worker无需设置`Alluxio master hostname`。
```
Comment (apc999) [inline docs/cn/Running-Alluxio-Fault-Tolerant.md]: <pre>`alluxio.master.hostname`</pre> please

File: docs/cn/Running-Alluxio-Fault-Tolerant.md (hunk 1 above)
Comment (MovingLi) [inline docs/cn/Running-Alluxio-Fault-Tolerant.md]: OK

File: docs/cn/Running-Alluxio-Fault-Tolerant.md (hunk 2 above)
Comment (MovingLi) [inline docs/cn/Running-Alluxio-Fault-Tolerant.md]: OK

**Comment by gpang** [comment]
Jenkins, add to whitelist

**Comment by apc999** [comment]
LGTM. thanks!

**Pull request merged**